code-review/README.md

```diff
@@ -31,7 +31,8 @@ Having Your Code Reviewed
 
 * Be grateful for the reviewer's suggestions. ("Good call. I'll make that
   change.")
-* Don't take it personally. The review is of the code, not you.
+* Be aware of [how hard it is to convey emotion online] and how easy it is to
+  misinterpret feedback.
```

Comment:
Maybe something like?

> A common axiom is "Don't take it personally. The review is of the code, not you."
> We used to include this, but now prefer to say what we mean:
> Be aware of [how hard it is to convey emotion online] and how easy it is to  misinterpret feedback.
> If a review seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent, in person if possible.